backup.diff

```diff
@@ -0,0 +1,145 @@
+diff --git a/hbase-it/src/test/java/org/apache/hadoop/hbase/chaos/actions/GracefulRollingRestartRsAction.java b/hbase-it/src/test/java/org/apache/hadoop/hbase/chaos/actions/GracefulRollingRestartRsAction.java
+index f4ef7978c9..f5458d1d54 100644
+--- a/hbase-it/src/test/java/org/apache/hadoop/hbase/chaos/actions/GracefulRollingRestartRsAction.java
++++ b/hbase-it/src/test/java/org/apache/hadoop/hbase/chaos/actions/GracefulRollingRestartRsAction.java
+@@ -63,7 +63,10 @@ public class GracefulRollingRestartRsAction extends RestartActionBaseAction {
+         rm.load();
+       } catch (Shell.ExitCodeException e) {
+         getLogger().info("Problem restarting but presume successful; code={}", e.getExitCode(), e);
++      } catch (Exception e) {
++        getLogger().info("Got Exception: " + e + ", continue");
+       }
++
+       sleep(RandomUtils.nextInt(0, (int)sleepTime));
+     }
+     getLogger().info("Enabling balancer");
+diff --git a/hbase-it/src/test/java/org/apache/hadoop/hbase/chaos/actions/RestartActionBaseAction.java b/hbase-it/src/test/java/org/apache/hadoop/hbase/chaos/actions/RestartActionBaseAction.java
+index ce41522fe7..d927055a3f 100644
+--- a/hbase-it/src/test/java/org/apache/hadoop/hbase/chaos/actions/RestartActionBaseAction.java
++++ b/hbase-it/src/test/java/org/apache/hadoop/hbase/chaos/actions/RestartActionBaseAction.java
+@@ -68,6 +68,8 @@ public abstract class RestartActionBaseAction extends Action {
+     sleep(sleepTime);
+     getLogger().info("Starting region server: {}", server);
+     startRs(server);
++    // Sleep 2 seconds to make sure RS is online.
++    sleep(2000);
```

Comment:
Make it configurable?

Reply:
Just figured that sleepTime is less than 1 second, so I think that is the perfect sleep time. Adding one more configuration is probably too much, what do you think? Thanks.

Reply:
Yeah, and I just realised we already have similar hardcoded sleep calls elsewhere in this code, so I guess this extra one is inline with that approach.